assets/differentialabundance_report.Rmd

```diff
@@ -577,6 +593,48 @@ print(p, vp=grid::viewport(gp=grid::gpar(cex=0.7)))
 
 ```
 
+### Outlier detection {.tabset}
+
+Outlier detection based on [median absolute deviation](https://wiki.arrayserver.com/wiki/index.php?title=CorrelationQC.pdf) was undertaken, the outlier scoring is plotted below.
+
+```{r, echo=FALSE, results='asis', warning=FALSE}
+
+for (iv in informative_variables){
```

Comment:
why not `apply()` instead of the for loop. Might not make a huge time difference here, but would still make a small difference I think.